_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,394 @@
+* * *
+
+## title: ONT FS.com
```

Comment:
I'd probably rename this to show the model as well, like https://raw.githubusercontent.com/hack-gpon/hack-gpon.github.io/main/_ont/ont-fs-com-gpon-onu-stick-with-mac.md

Also, it's not showing up in the sidebar

Reply:
fixed